package/test/daml/Daml.Finance.Instrument.Bond.Test/daml.yaml

```diff
@@ -1,14 +1,14 @@
 # Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-sdk-version: 2.10.0-snapshot.20241206.13126.0.vf0f16532
+sdk-version: 2.10.0-snapshot.20241106.0
 name: daml-finance-instrument-bond-test
 source: daml
 version: 1.0.0
 dependencies:
   - daml-prim
   - daml-stdlib
-  - daml-script
+  - daml-script-beta
```

Comment:
What's the difference between `daml-script-beta` and `daml-script`?

Reply:
It as needed when building using LF 1.17, and has now been renamed to `daml-script-lts` going forward.